impl/torch/functions/functions_ext.cpp

```diff
@@ -236,15 +243,15 @@ diopiError_t diopiFlashAttention(diopiContextHandle_t ctx, diopiTensorHandle_t a
                                  float p_dropout, float softmax_scale, bool is_causal, int32_t window_size_left, int32_t window_size_right) {
     impl::aten::setCurStream(ctx);
 
-    // handle param[out]
+    // handle param[out]mha_bwd
     DIOPI_IMPL_BUILD_ATEN_OPTIONAL(optOut, attention_out);
+    DIOPI_IMPL_BUILD_ATEN_OPTIONAL(linearBiasSlopes, alibi_slopes);
 
     // handle param[in]
     auto atGen = buildGeneratorForMha(ctx, gen, p_dropout);
     auto atQ = impl::aten::buildATen(q);
     auto atK = impl::aten::buildATen(k);
     auto atV = impl::aten::buildATen(v);
```

Comment:
这是个 in param，还有命名一致性
```suggestion

    // handle param[in]
    auto atGen = buildGeneratorForMha(ctx, gen, p_dropout);
    auto atQ = impl::aten::buildATen(q);
    auto atK = impl::aten::buildATen(k);
    auto atV = impl::aten::buildATen(v);
    DIOPI_IMPL_BUILD_ATEN_OPTIONAL(optLinearBiasSlopes, alibi_slopes);
```